src/dictionary.c

```diff
@@ -89,6 +99,7 @@ int dict_chars_update(dict_t *d, char *str) {
     int len = strnlen(str, MAXSTRLEN);
 
     if (str[len] != '\0') {
+        log_trace("returning EXIT_FAILURE");
```

Comment:
these logging comments need to be more specific so we know which functions are printing them

Reply:
done